pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/route/PatchMedicalProviderRoute.java

```diff
@@ -66,6 +66,7 @@ public Object handle(
                     String physicianName;
                     String city;
                     String state;
+                    String country;
                     try {
                         JsonElement data = new JsonParser().parse(request.body());
                         if (!data.isJsonObject() || data.getAsJsonObject().entrySet().size() == 0) {
```

Comment:
Any API changes must also have their associated documentation updated in `pepper-apis/docs/specification`.